tabular/tests/unittests/resource_allocation/test_total_resource_allocation.py

```diff
@@ -190,7 +210,7 @@ def test_nonbagged_model_with_total_resources_and_model_resources(mock_system_re
             hyperparameters={
                 'ag_args_fit': {
                     'num_cpus': 1,
-                    'num_gpus': 1
+                    'num_gpus': 0
```

Comment:
Why is this hard-coded? Wouldn't it be way easier to just have this be in the arguments of the test?